src/reference-designs/tkg-edge.md

```diff
@@ -0,0 +1,885 @@
+# VMware Tanzu Edge Solution Architecture
+
+Edge computing is gaining a lot of momentum for a variety of reasons that range from avoidance of latency, data volume, and bandwidth considerations to need for autonomous disconnected operation, privacy and security. These reasons are pushing businesses across all industries to run more and more applications at edge locations. Traditionally, these applications were run in virtual machines, but with emergence of Kubernetes and microservices architectures, more applications are being containerized and there is a need for running a cloud native compute stack at the edge.
+
+For those companies that already have remote offices and branch offices with a few servers running edge workloads, running modern, distributed applications across your edge locations can add a new layer of complexity. There is a significant difference between running a cloud native platform at the core data center versus running it at edge locations.
+
+The VMware Edge Compute Stack helps companies manage their edge locations with little or no local IT staff and enables them to run VM workloads at these sites. Now with the addition of VMware Tanzu, you can simplify how you architect and deploy a cloud native stack at hundreds or thousands of edge locations so that you can run cloud native applications at the edge.
+
+The Tanzu Design for VMware Edge Compute Stack described here offers a best practice for running Tanzu at edge locations where the VMware Edge Compute Stack is already deployed. We’ll specifically address these challenges at the edge:
```

Comment:
The Tanzu design ...deployed. We will specifically address the following challenges ...